### ISSUE TITLE
Add Windows Terminal on the Win+X menu

### ISSUE THREAD

**Issue by SJang1** [microsoft/terminal#1421]
# Summary of the new feature/enhancement

Like #1060 but

1. just starting windows terminal. (not starting from the directory I am in.)
2. add at windows start button right-click context menu, not explorer right-click context menu.
(Seems to be likely under Windows Powershell or replace Windows Powershell button.)

![image](https://user-images.githubusercontent.com/29011440/59965770-923cf680-954d-11e9-95a9-ad56dca65cfe.png)
_Sorry that the language is not english, because of windows bug, I couldn't change language at this build of windows._

**Comment by arseny92** [comment]
off: inability to change system display language is not a bug for CoreSingleLanguage system SKU.

**Comment by SJang1** [comment]
@arseny92 I just can't get to language setting menu by the bug.

**Comment by markbarnes** [comment]
I agree. I also would like to see an option to have Terminal in the Power User menu.

**Comment by DHowett-MSFT** [comment]
There's a magic folder that contains shortcuts that show up in the Win+X menu, actually. It's somewhere in `%APPDATA%\Microsoft`!

**Comment by DHowett-MSFT** [comment]
We're tackling this in a different way; see #1466

**Comment by arseny92** [comment]
> There's a magic folder that contains shortcuts that show up in the Win+X menu, actually. It's somewhere in `%APPDATA%\Microsoft`!

Just `%APPDATA%` is for Appdata\Roaming

The folder is at `%LOCALAPPDATA%\Microsoft\Windows\WinX` (Not recommended to touch default groups, to not mess menu order). Just do a `Group4` folder, put in a shortcut to `shell:AppsFolder\Microsoft.WindowsTerminal_8wekyb3d8bbwe!App`

CMD/Powershell Taskbar Settings toggle sets dword at `HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced` , `DontUsePowerShellOnWinX` and is pretty much not useful for this case

**Comment by Karl-WE** [comment]
> > There's a magic folder that contains shortcuts that show up in the Win+X menu, actually. It's somewhere in `%APPDATA%\Microsoft`!
> 
> Just `%APPDATA%` is for Appdata\Roaming
> 
> The folder is at `%LOCALAPPDATA%\Microsoft\Windows\WinX` (Not recommended to touch default groups, to not mess menu order). Just do a `Group4` folder, put in a shortcut to `shell:AppsFolder\Microsoft.WindowsTerminal_8wekyb3d8bbwe!App`
> 
> CMD/Powershell Taskbar Settings toggle sets dword at `HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced` , `DontUsePowerShellOnWinX` and is pretty much not useful for this case. If only we could override that...


There is a tool for this but beware the site is full of ads and lingering links
https://winaero.com/comment.php?comment.news.30
Download: https://winaero.com/request.php?21

**Comment by mikemaccana** [comment]
For an alternative idea, check out  #1594 - which is about keeping the existing items in Win+X, but making them open using a preferred terminal app, that could be set to Windows Terminal (much like we set preferred web browser or preferred email client).

**Comment by zadjii-msft** [comment]
Or possibly #492, which is what I think @DHowett-MSFT meant

**Comment by BaconTentacles** [comment]
> The folder is at `%LOCALAPPDATA%\Microsoft\Windows\WinX` (Not recommended to touch default groups, to not mess menu order). Just do a `Group4` folder, put in a shortcut to `shell:AppsFolder\Microsoft.WindowsTerminal_8wekyb3d8bbwe!App`

I tried doing the above, because that sounded awesome.  Created a Group4 folder, and attempted to add some shortcuts.  Noticed a couple of things straightaway:

1. Attempting to paste the shell:AppsFolder link into a shortcut (using right-click | New | Shortcut) results in the message that the file cannot be found.  That was on my home PC.  On my corporate PC, I think it said something to the effect that I couldn't use that file type or some such.  Both PC's are running Windows 10 1909.  
2. So, I went to shell:AppsFolder via Win-R, found terminal, right-clicked and saved a shortcut to the desktop, then copied that in.  
3. Added a shortcut to Notepad for good measure.  Just to see if something would show up.
4. Rebooted.  Logged back in.  Hit Win-X.  Absolutely no change.  Nothing shows up from "Group4".  Does this no longer work, or some such?  

I am reluctant to try the WinAero thing, but I guess I can give that a go.
